zookeeper-server/src/test/java/org/apache/zookeeper/server/backup/BackupConfigTest.java

```diff
@@ -146,13 +146,13 @@ public void testRetentionPeriod() throws Exception {
 
   @Test
   public void testRetentionMaintenanceInterval() throws Exception {
-    assertEquals(BackupConfig.DEFAULT_RETENTION_MAINTENANCE_INTERVAL_MS,
+    assertEquals(BackupConfig.DEFAULT_RETENTION_MAINTENANCE_INTERVAL_HOURS,
         builder().build().get().getRetentionMaintenanceInterval());
-    long expected = 3 * 60 * 60 * 1000L; // 3 hours
-    assertEquals(expected, builder().setRetentionMaintenanceInterval(expected).build().get()
+    int expected = 3; // 3 hours
+    assertEquals(expected, builder().setRetentionMaintenanceIntervalInHours(expected).build().get()
         .getRetentionMaintenanceInterval());
     assertEquals(expected, builder()
-        .withProperty(BackupSystemProperty.BACKUP_RETENTION_MAINTENANCE_INTERVAL_MS, "10800000")
+        .withProperty(BackupSystemProperty.BACKUP_RETENTION_MAINTENANCE_INTERVAL_HOURS, "10800000")
```

Comment:
Since we changed the unit, shall we change the value to be compatible with original test purpose?

Reply:
Good catch!